changelog/README.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 9th January 2025
+* Extended [Get all enterprises](../operations/enterprises.md#get-all-enterprises) response object with `Address`.
+* Removed deprecation of `Address` in [Enterprise](../operations/configuration.md#enterprise), this affects the following operations:

```

Comment:
Preferred format:
* [Get configuration](../operations/configuration.md#get-configuration): Removed deprecation of `Address` in [Enterprise](../operations/configuration.md#enterprise).

Reply:
I wonder if we should go further and add "`Address` no longer deprecated.", just to make it crystal clear. What do you think?

Reply:
@MikeAdamsMews what about combination?
* [Get configuration](../operations/configuration.md#get-configuration): 
  * `Address` is no longer deprecated in [Enterprise](../operations/configuration.md#enterprise).